auth-web/src/composables/transactions-factory.ts

```diff
@@ -73,6 +73,24 @@ export const useTransactions = () => {
       if (response?.data) {
         transactions.results = response.data.items || []
         transactions.totalResults = response.data.total
+        const transactionClone = [...transactions.results]
+        transactionClone.forEach((transaction: Transaction, i: number) => {
+          if (transaction.refundDate) {
```

Comment:
Is this too broad? there could be multiple states that the refundDate would be set.